header.php

```diff
@@ -30,11 +30,13 @@
 
 		<?php if ( is_front_page() ) : ?>
 			<div class="site-logo">
-
-				<a class="HMLogo HMLogo-White" href="<?php echo esc_url( home_url() ); ?>/">
-					Human Made Limited
-				</a>
-
+				<?php if ( function_exists( 'the_custom_logo' ) ) {
```

Comment:
has_custom_logo .. yes but  docuemntation says that you shoudl check for old versionf o WordPress by checking if the fucntion exists too so i just added that bit. I dunno if you wanted to hve  aminimum WordPress version requirement.